playbooks/tests/tasks/lbaas.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Should we be moving more towards using rally as the test driver (by the way, I'm close to having rally running against our Ursula-deployed test cloud)?  I'm not sure what's supported re: v2 in rally, if anything, but if there are things that are missing, maybe it's time better spent working upstream / adding it to rally rather than writing ansible playbooks to test features?